resolwe_bio/processes/variant_calling/vc_chemut.yml

```diff
@@ -174,6 +176,7 @@
 
         echo "sorting, marking duplicates, indexing"
         java \
+          -Xmx16g \
```

Comment:
Use the memory limit specified in resources.

Reply:
Yes, isn't that what I've done here?

Reply:
@dblenkus, please elaborate.